frontend/src/game/StartPanel.jsx

```diff
@@ -40,21 +42,41 @@ const StartControls = () => {
 };
 
 const Options = () => {
-  const {useTimer} = App.state;
-  const timers = ["Fast", "Moderate", "Slow", "Leisurely"];
```

Comment:
Reviewing on my phone but I think you still need this

Reply:
It's in the initialState in the reducer.